### ISSUE TITLE
Show transfer speed (not just progress) and ETA

### ISSUE THREAD

**Issue by ihubgit** [Jacalz/rymdport#100]
### Checklist

- [X] I have searched the [issue tracker](https://github.com/Jacalz/rymdport/issues) for open issues that relate to the same feature, before opening a new one.
- [X] This issue only relates to a single feature. I will open new issues for any other features.

### Is your feature request related to a problem?

Hi, i appreciate being able to see the progress of the transfer, but i'd like to be able to see the transfer speed itself, so i can quality check the  speed of the transfer (and compare it with other applications or file hosters for that matter).
It would also be nice to have a clue of the estimation of the end of the transfer.

### Describe the solution you'd like to see.

So please add a tab indicating the current transfer speed (a simple 6.4MB/s f.ex. will do).
And another tab for the estimated time remaining for the end of the transfer ( (like ending in 00:15:25).

**Comment by Jacalz** [comment]
Thanks for the suggestion. This sounds like a good idea. Do you have any ideas for how this could be presented to the user in a good way? I’m a bit unsure where to put it to be honest

**Comment by ihubgit** [comment]
you could just add it before (or after) the progress bar f.ex. as extra columns.

**Comment by Jacalz** [comment]
I'm not sure if that would look very good unfortunately and it would become unnecessary information has soon as the transfer has completed

**Comment by ihubgit** [comment]
but that's also true for the other information once the transfer is finished no ?
Alternatively, you may want to display it as a pop up when hovering over the progress bar f.ex. but i suppose that's harder to code than an extra column.
If you make those extra columns an option for the end user, he can decide whether to display them or not.

**Comment by Jacalz** [comment]
I'm not that sure about extra columns and hiding them. I'm a bit worried that it will start looking like an Excel sheet. Perhaps showing a small pop-up when hovering the progress bar could be a good idea?

**Comment by ihubgit** [comment]
yea that's fine with me.

**Comment by Jacalz** [comment]
I plan on simplifying the application as part of a big rewrite in v4.0.0. This will likely not make it into the new design. Closing for now.
